catalog/app/components/FileEditor/QuiltConfigEditor/QuiltSummarize/QuiltSummarize.tsx

```diff
@@ -0,0 +1,782 @@
+                  />
+                }
+                labelPlacement="start"
+                label="Expand"
+                title="Whether preview is expanded by default or not"
+              />
+              <M.FormGroup>
+                {row.columns.length > 1 && (
+                  <M.TextField
+                    disabled={disabled}
+                    label="Width"
+                    name="width"
+                    onChange={(event) =>
+                      onChangeValue('width', event.currentTarget.value)
+                    }
+                    value={file.width || ''}
+                    fullWidth
+                    className={classes.field}
+                    size="small"
+                    helperText="Width in pixels or percents"
+                  />
+                )}
+
+                <M.FormControl className={classes.field} fullWidth size="small">
+                  <M.InputLabel>Renderer</M.InputLabel>
+                  <M.Select
+                    className={classes.select}
+                    value={file.type?.name || ''}
+                    onChange={(event) =>
+                      onChangeType('name', event.target.value as Summarize.TypeShorthand)
+                    }
+                  >
+                    <M.MenuItem value="">
+                      <i>Default</i>
+                    </M.MenuItem>
+                    {schema.definitions.typeShorthand.enum.map((type) => (
+                      <M.MenuItem key={type} value={type}>
+                        {type}
+                      </M.MenuItem>
+                    ))}
+                  </M.Select>
+                </M.FormControl>
+
+                {file.type && (
+                  <M.TextField
+                    disabled={disabled}
+                    label="Height"
+                    name="height"
+                    onChange={(event) =>
+                      onChangeType('style', { height: event.currentTarget.value })
+                    }
+                    value={file.type.style?.height || ''}
+                    fullWidth
+                    className={classes.field}
+                    size="small"
+                    placeholder="Ex., 1000px"
+                    helperText="Height as an absolute value (in `px`, `vh`, `em` etc.)"
+                  />
+                )}
+
+                {file.type?.name === 'perspective' && (
+                  <M.TextField
+                    disabled={disabled}
+                    label="Perspective config"
+                    name="config"
+                    onChange={(event) => {
+                      try {
+                        onChangeType('config', JSON.parse(event.currentTarget.value))
+                      } catch (error) {
+                        Log.error(error)
+                      }
+                    }}
+                    helperText="Restores renderer to a state previously returned by saving config. Should be valid JSON object"
```

Comment:
maybe smth like this?
```suggestion
                    helperText="Restores renderer state using a previously saved configuration. Configuration must be a valid JSON object."
```